tests/features/eulogin/eulogin.feature

```diff
@@ -37,6 +42,13 @@ Feature: Log in through EU Login
 
     Given I select the radio button "I am a new user (create a new account)"
     When I press "Next"
+
+    Then I should see the warning message "You must accept this agreement before continuing."
+    But I should not see the success message "Fill in the fields below to let the Joinup community learn more about you!"
+
+    Given I check "I have read and accept the Legal notice"
```

Comment:
This is an action taken by the user that is part of the test steps, so this should have `When` instead of `Given`. 

`Given` is only used for steps that are setting up the condition of the test environment, similar to the `setUp()` method in a unit test.